frontend/alert/src/components/Toast.js

```diff
@@ -64,52 +66,53 @@ const RightItem = styled.div`
 
 export const ToastType = Object.freeze({ Success: 1, Warning: 2, Error: 3 });
 
-export const Toast = ({ type, course }) => {
+const Toast = ({
+    onClose, children, isSuccess, isWarning, isError, type,
```

Comment:
Wasn't sure what isSuccess, isWarning and isError is for if they have the same information as the type enum

Reply:
Not used for anything atm... I'll take them out. The idea was that it would be nicer to type <Toast isSuccess /> if you were declaring a toast than it would be to do the enum, but it's also kind of confusing